### PR TITLE
feat(active-release): Add suspect releases to issue details

### PR DESCRIPTION
**Pull request by scttcper**
Behind the `active-release-monitor-alpha` flag. Adds suspect releases to the issue details sidebar as part of active release

Screen Shot 2022-07-29 at 5 21 23 PM

### CODE REVIEW

File: static/app/components/group/seenInfo.tsx [hunk 1]
```diff
@@ -33,13 +33,6 @@ class SeenInfo extends Component<Props> {
     return release?.version !== nextProps.release?.version || date !== nextProps.date;
   }
 
-  getReleaseTrackingUrl() {
```
Comment (scttcper) [inline static/app/components/group/seenInfo.tsx:36]: just happened to see something unused while in this folder

File: static/app/components/group/suspectReleases.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+type Props = AsyncComponent['props'] & {
+  group: Group;
+};
+
+type State = AsyncComponent['state'] & {
+  suspectReleases: {suspectReleases: Release[]} | null;
+};
+
+class SuspectReleases extends AsyncComponent<Props, State> {
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
```
Comment (evanpurkhiser) [inline static/app/components/group/suspectReleases.tsx:24]: wanna try using `useApiRequests` here?

File: static/app/components/group/suspectReleases.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+import TimeSince from '../timeSince';
+
+import SidebarSection from './sidebarSection';
+
+type Props = AsyncComponent['props'] & {
+  group: Group;
+};
+
+type State = AsyncComponent['state'] & {
+  suspectReleases: {suspectReleases: Release[]} | null;
```
Comment (evanpurkhiser) [inline static/app/components/group/suspectReleases.tsx:20]: Annoying how this nests. Is this how we designed the API?

File: static/app/components/group/suspectReleases.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    const {group} = this.props;
+    return [['suspectReleases', `/issues/${group.id}/suspect-releases/`]];
+  }
+
+  getDefaultState(): State {
+    return {
+      ...super.getDefaultState(),
+      suspectReleases: null,
+    };
+  }
```
Comment (evanpurkhiser) [inline static/app/components/group/suspectReleases.tsx:34]: Do you actually need this?

File: static/app/components/group/suspectReleases.tsx (hunk 3 above)
Comment (scttcper) [reply static/app/components/group/suspectReleases.tsx]: flattened it in https://github.com/getsentry/sentry/pull/37396

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**